packages/host/src/brain.rs

```diff
@@ -171,15 +185,21 @@ impl Brain {
 
                 display: Display::new(peripherals_tx.clone(), display_rx),
             }),
+            link_addr: link_addr_2.clone(),
```

Comment:
Does this need to be cloned twice?

Reply:
Fixed in https://github.com/vexide/vex-v5-qemu/pull/35/commits/e62bf1ddc7d842783810b277553c57f48521a56d